### ISSUE TITLE
HTTP redirection

### ISSUE THREAD

**Issue by eed3si9n** [foundweekends/conscript#99]
Branching out from #69 /cc @michalmela 

### steps

```
$ cs n8han/giter8
An unexpected error occurred: Please check your internet connection.
You may need to generate a github access token.
see https://help.github.com/articles/creating-an-access-token-for-command-line-use/
Exception message: Unexpected response status: 301
```

### expectation

HTTP client should follow redirect on 301 if possible

**Comment by jerzygangi** [comment]
A note to future Googlers:

If you come here after having run `n8han/giter8`, you need to instead run `foundweekends/giter8` since the repository was moved.

**Comment by michalmela** [comment]
Due to https://github.com/foundweekends/conscript/pull/102#issuecomment-228577889 , HTTP client should now follow redirects, so `cs n8han/giter8` works again.

**Comment by eed3si9n** [comment]
Fixed by #102
